python/cudnn/gemm/frost/kernel_templates/sm100_matmul_2ctamma_static.py

```
@@ -592,25 +651,21 @@ def _kernel(
-                    if tail_stage == acc_stages:
-                        tail_stage = cutlass.Int32(0)
-                        tail_phase = tail_phase ^ 1
-                    while not nvvm.mbarrier_try_wait_parity(
-                        acc_empty_mbar_ptr.subview(tail_stage),
-                        tail_phase,
-                        time_limit=10_000_000,
-                    ):
-                        pass
-            nvvm.bar_warp_sync(0xFFFFFFFF)
-
+            for _ in range(acc_stages):
```

Ruff (0.16.1)

[error] Undefined name `acc_stages`

(F821)